storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -11,21 +11,51 @@ const meta = {
   component: Heading,
   args: {
     children: 'Jouw typograaf biedt mij zulke exquise schreven!',
-    level: 1,
-    size: 'level-1',
+    level: undefined,
+    size: undefined,
+    inverseColor: false,
   },
   argTypes: {
+    level: {
+      control: {
+        type: 'number',
+        min: 1,
+        max: 4,
+      },
+    },
     size: {
+      control: 'radio',
       options: ['level-1', 'level-2', 'level-3', 'level-4'],
     },
+    inverseColor: { control: 'boolean' },
   },
 } satisfies Meta<typeof Heading>
 
 export default meta
 
 type Story = StoryObj<typeof meta>
 
-export const Heading1: Story = {}
+export const Default: Story = {
+  decorators: [
+    (Story, context) => (
+      <div
+        style={{
+          background: context.args.inverseColor ? '#004699' : 'transparent',
+          padding: context.args.inverseColor ? '16px' : 0,
```

Comment:
This prevents a layout change, and the `px` aren’t necessary:

```suggestion
          margin: context.args.inverseColor ? -16 : 0,
          padding: context.args.inverseColor ? 16 : 0,
```